File: exercicios/R/1142/1142.r
input = file('stdin','r')

x = as.integer(1)

n = as.integer(readLines(input, n=1))

for(i in 1:n){
    write(sprintf("%d %d %d PUM",x,x+1,x+2),'')
    x = x + 4
}

File: exercicios/JavaScript/1142/1142.js
var input = require('fs').readFileSync('stdin','utf8');
var lines = input.split('\n');

var n = parseInt(lines.shift()), x=1;

for(let i=0; i<n; i++){
    console.log(x + " " + (x+1) + " " + (x+2) + " PUM");
    x += 4;
}